src/Controller/PagesController.php

```diff
@@ -42,7 +42,9 @@ public function display(...$path)
     {
         $count = count($path);
         if (!$count) {
-            return $this->redirect('/');
```

Comment:
But redirect returns a response. In the future we'll actually require `return $this->redirect()`.